src/calculateRentalCost.js

```diff
@@ -29,6 +29,9 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const sum = days * 40;
+
+  return days >= 7 ? sum - 50 : days >= 3 ? sum - 20 : sum;
```

Comment:
Use if else or switch case here. This operation is quite hard to read. We don't write like that on real projects.
Also don't use magic numbers. Put all the numbers in variables